sensor/mcp9808/mcp9808.go

```diff
@@ -0,0 +1,484 @@
+// Package mcp9808 is a driver for the MCP9808 temperature sensor
+// and all code is based off of the documentation found here:
+// http://ww1.microchip.com/downloads/en/DeviceDoc/25095A.pdf
+package mcp9808
+
+import (
+	"log"
+	"sync"
+	"time"
+
+	"github.com/kidoman/embd"
+)
+
+const (
+	// default I2C address for device
+	address = 0x18
+
+	// Register addresses.
+	regConfig       = 0x01
```

Comment:
on second thought, since the values are not inherently sequential, maybe what you have is better...